packages/checkbox-group/theme/lumo/vaadin-checkbox-group-styles.js

```diff
@@ -35,7 +35,7 @@ const checkboxGroup = css`
     -webkit-text-fill-color: var(--lumo-disabled-text-color);
   }
 
-  :host([focused]:not([disabled])) [part='label'] {
```

Comment:
To be sure, this style was not applied anyway because there is no way to focus a disabled checkbox group?

Reply:
Yes, exactly. I think adding it to the selector was a mistake.